aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -184,6 +191,11 @@ at::Tensor& random_from_to_impl(at::Tensor& self, int64_t from, c10::optional<in
 template<template<typename> class normal_kernel, typename RNG>
 Tensor& normal_impl_(Tensor& self, double mean, double std, c10::optional<Generator> gen) {
   CHECK_NORMAL_STD(std);
+
+  // return earlier for not invoking normal_kernel
+  // see https://github.com/pytorch/pytorch/issues/103418 for more details
```

Comment:
A bit weird to only leave the comment in `normal_impl_` while this logic applies to other functions here too. Maybe revise and move the comment to the definition of `CHECK_EMPTY_AND_RETURN`?